Description: Add loong64 support
Author: Han Gao <gaohan@iscas.ac.cn>

Last-Update: 2023-05-14

--- /dev/null
+++ golang-github-boltdb-bolt-1.3.1/bolt_loong64.go
@@ -0,0 +1,12 @@
+// +build loong64
+
+package bolt
+
+// maxMapSize represents the largest mmap size supported by Bolt.
+const maxMapSize = 0x8000000000 // 512GB
+
+// maxAllocSize is the size used when creating array pointers.
+const maxAllocSize = 0x7FFFFFFF
+
+// Are unaligned load/stores broken on this arch?
+var brokenUnaligned = false
